pkg/controller/ipam/validate_test.go

```diff
@@ -320,10 +320,22 @@ func TestEgressControllerValidateExternalIPPool(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			stopCh := make(chan struct{})
+			defer close(stopCh)
+
+			namespace, pool, statefulSet := initTestObjects(false, false, 0)
+			controller := newFakeAntreaIPAMController(pool, namespace, statefulSet)
+			controller.informerFactory.Start(stopCh)
+			controller.crdInformerFactory.Start(stopCh)
+			controller.informerFactory.WaitForCacheSync(stopCh)
+			controller.crdInformerFactory.WaitForCacheSync(stopCh)
+
+			go controller.Run(stopCh)
```

Comment:
is this change actually needed now that we dropped the check for range overlap between IPPools?

Reply:
done